examples/hybrid/sphere/deformation_flow.jl

```diff
@@ -197,19 +201,19 @@ function rhs!(dydt, y, (coords, face_coords), t)
     cuvw = Geometry.Covariant123Vector.(uₕ) .+ Geometry.Covariant123Vector.(cw)
 
     @. dρq1 -= hdiv(cuvw * ρq1)
-    @. dρq1 -= vdivf2c(w * Ic2f(ρq1))
+    @. dρq1 -= ρ * vdivf2c(third_order_upwind_c2f.(w, ρq1 ./ ρ))
```

Comment:
Ah, it should go inside the divergence
```suggestion
    @. dρq1 -= vdivf2c(Ic2f(ρ) * third_order_upwind_c2f.(w, ρq1 ./ ρ))
```

Reply:
of course!

Reply:
That results in 
```
ERROR: LoadError: MethodError: no method matching Jcontravariant3(::Float64, ::ClimaCore.Geometry.LocalGeometry{(1, 2, 3), ClimaCore.Geometry.LatLongZPoint{Float64}, Float64, StaticArrays.SMatrix{3, 3, Float64, 9}})
Closest candidates are:
  Jcontravariant3(::ClimaCore.Geometry.AxisTensor, ::ClimaCore.Geometry.LocalGeometry) at ~/CliMA/ClimaCore.jl/src/Geometry/conversions.jl:157
Stacktrace:
  [1] stencil_left_boundary
```
I will now try to just put the `Ic2f(ρ) ` bit inside the `vdivf2c` operator